home/tests/test_migrations.py

```diff
@@ -53,32 +53,22 @@ def test_deduplicates_orderedcontentset_slugs(self) -> None:
         default_locale = Locale.objects.get(language_code="en")
         pt_locale, _created = Locale.objects.get_or_create(language_code="pt")
 
-        # Create ordered content sets with duplicate slugs
-        OrderedContentSet.objects.create(
-            name="first", slug="duplicate", locale=default_locale
-        )
-        OrderedContentSet.objects.create(
-            name="second", slug="duplicate", locale=default_locale
-        )
-        OrderedContentSet.objects.create(
-            name="first_pt", slug="duplicate", locale=pt_locale
-        )
-        OrderedContentSet.objects.create(
-            name="third", slug="duplicate-2", locale=default_locale
-        )
-        OrderedContentSet.objects.create(
-            name="unique", slug="unique", locale=default_locale
-        )
+        # Create ordered content sets with blank slugs
+        OrderedContentSet.objects.create(name="first", slug="", locale=default_locale)
+        OrderedContentSet.objects.create(name="first", slug="", locale=default_locale)
+        OrderedContentSet.objects.create(name="first_pt", slug="", locale=pt_locale)
+        OrderedContentSet.objects.create(name="third", slug="", locale=default_locale)
+        OrderedContentSet.objects.create(name="unique", slug="", locale=default_locale)
```

Comment:
Maybe we can have some of the names here with whitespace and repeated dashes, to ensure that we're converting name -> slug correctly?

Reply:
And mixed case?

Reply:
slugify lowercases